src/clients/UBAClient/UBAClientUtilities.ts

```diff
@@ -668,18 +680,20 @@ export async function refundRequestIsValid(
   // refund was sent.
   const fill = destSpoke.getFillsForRelayer(relayer).find((fill) => {
     // prettier-ignore
```

Comment:
note to self - delete this.

Reply:
Done: https://github.com/across-protocol/sdk-v2/pull/501/commits/f7cc09614b72bcf31430469d6d25cf71fbe766b1.